wp-content/themes/humanity-theme/includes/blocks/header/class-header-block-renderer.php

```diff
@@ -176,7 +184,7 @@ protected function image() {
 	 * @return void
 	 */
 	protected function metadata() {
-		if ( ! $this->image->id() ) {
+		if ( ! $this->image->id() || ! $this->video->id() ) {
```

Comment:
If the block has an image but no video, or vice versa, no metadata will be output.
This should be an `&&` operator instead of an `||`.

Reply:
resolved